src/providers/etherscan.js

```diff
@@ -1,29 +1,95 @@
+import rp from 'request-promise';
+import logger from '../logger';
+import { getAccessToken } from '../config';
+
+const path = require('path');
+
+const fs = (require('fs'));
+
+const expectedResponce = {
+  OK: 'OK',
+};
+
 /**
- * 
- * This is private function 
- * that saves the file ABI object into file locally
+ *  Validate if Etherscan response was succssefful should
+ *  throw if a reponse had an error
+ *  @param reponse: object
  *
- * @param address
- * return: string 
  */
-export const saveABIFile = data => 'FILE_PATH'; // eslint-disable-line no-unused-vars
-
+const ValidateResponce = (response) => {
```

Comment:
Suggestion: Since you just have one argument you can take the parenthesis off

Reply:
That is automatically fixed by ESlint